index.js

```diff
@@ -1 +1,11 @@
-var root = document.querySelector('#root');
\ No newline at end of file
+var root = document.querySelector('#root');
+// image we have four compnets that we have from source directory 
```

Comment:
self-documenting code is better than comments! Especially if the comments have spelling errors.

Please remove these comments.